pysheaf/pysheaf.py

```diff
@@ -893,6 +931,9 @@ def fuseAssignment(self,assignment, activeCells=None, testSupport=None, method='
                         num_generations - the number of iterations that the genetic algorithm runs
                         num_ele_Hallfame - the number of top individuals that should be reported in the hall of fame (hof)
         """
+        if activeCells != None and not (set(testSupport) <= set(self.maxTestSupport(activeCells))):
```

Comment:
It was my intention that testSupport and activeCells could be completely unrelated.  That might be cause for a WARNING rather than an ERROR